src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -159,6 +164,28 @@ const AuthoritiesSearch = ({
 
   const [storedFilterPaneVisibility] = useLocalStorage(filterPaneVisibilityKey, true);
   const [isFilterPaneVisible, setIsFilterPaneVisible] = useState(storedFilterPaneVisibility);
+  const [instancesQuickExportInProgress, setInstancesQuickExportInProgress] = useState(false);
```

Comment:
Let's move this variable to a separate hook `useAuthorityExport` - I've described this hook later

Reply:
Also, should be named `authotitiesQuickExportInProgress` because it's related to authority records and not instance records